src/serializer.ts

```diff
@@ -110,6 +113,32 @@ export class DocxSerializerState<S extends Schema = any> {
     });
   }
 
+  renderReference(parent: ProsemirrorNode<S>) {
```

Comment:
Current thinking at end of day here is that we should have a single method `renderReferences` which serializes the references and also the title `References` into the state